librecad/src/actions/rs_actionmodifyround.cpp

```diff
@@ -80,11 +81,16 @@ void RS_ActionModifyRound::trigger() {
         deletePreview();
 
         RS_Modification m(*container, graphicView);
+		RS_Entity* radius = nullptr;
+		if (changeRadius) {
+			radius = RS_Information::getRadiusArc(entity1, entity2);
+		}
 		m.round(pPoints->coord2,
 				pPoints->coord1,
-                (RS_AtomicEntity*)entity1,
+				(RS_AtomicEntity*)entity1,
```

Comment:
minor coding style: try to avoid C-style casting, here entity1 and entity2 are both AtomicEntities, so static_cast is preferred over C-Style